packages/react-scripts/scripts/audit.js

```diff
@@ -0,0 +1,75 @@
+// @remove-file-on-eject
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+'use strict';
+
+const { createServer } = require('http');
+const { writeFileSync } = require('fs');
+const { join } = require('path');
+const { choosePort } = require('react-dev-utils/WebpackDevServerUtils');
+const open = require('open');
+const handler = require('serve-handler');
+const lighthouse = require('lighthouse');
+const chromeLauncher = require('chrome-launcher');
+const paths = require('../config/paths');
+
+const DEFAULT_PORT = parseInt(process.env.PORT, 10) || 3000;
+const HOST = process.env.HOST || '0.0.0.0';
+
+// https://github.com/GoogleChrome/lighthouse/blob/master/docs/readme.md#using-programmatically
+const launchChromeAndRunLighthouse = (url, opts) => {
+  return chromeLauncher
+    .launch({ chromeFlags: opts.chromeFlags })
+    .then(chrome => {
+      opts.port = chrome.port;
+      return lighthouse(url, opts).then(results => {
```

Comment:
async/await may make it a bit simpler to cover failure cases such as any of this process throwing in which case we should probably clean up the browser instance.

Reply:
Agreed. I just left this as per the docs for now - and forgot that we don't support Node 6.